src/en/wizden-staff/maintainer/maintainer-workgroup-policy.md

```diff
@@ -0,0 +1,113 @@
+
+### Joining
+The joining procedure is semiformal:
+1. A Maintainer contacts a member of the group and asks to join. A member of the group can also nominate a person to join the group.
+2. A vote must be created, for three days, in the group's Discord channel dedicated to the workgroup.
+3. A supermajority (66%) of votes in favor must be observed in order for the vote to pass.
+   - Voting can be expedited if a unanimous decision presents itself.
+   - If there is no clear majority, the vote must remain open for the full 3 days.
+
+### Removal
+Removal is entirely handled by Lead Maintainers, Project Managers, or Wizards.
+Members of the group that are inactive for a long time (~6 months) or do not frequent discussions should be removed.
+
+### General Discussions
+Members of the group are encouraged to be transparent, and discussions should partake in a relatively public channel.
+These channels can be restricted to contributors but shouldn't be hidden away in internal channels unless sensitive information is being discussed (a game-breaking exploit or otherwise).
+
+Generally, active public discussion can help give insight to contributors as to what needs to be worked on and what can be made better in that topic.
+Development can be coordinated between maintainers and contributors, which allows maintainers to spend more time reviewing desirable pull requests instead of starting votes on undesirable ones.
+
+### PR Reviews
+Any pull request to a game area controlled by a maintainer workgroup must receive one conceptual approval from a Maintainer belonging to that workgroup.
+Pull requests authored by a workgroup member that target a workgroup cannot be self-approved for design.
+
+This is to encourage directing pull requests targeting certain game areas to groups that have more experience with that game area.
+
+If a pull request belonging to a maintainer workgroup is closed citing game design reasons, these reasons **must** be citable from any merged design document.
+If a reason cannot be cited from any merged design document, the design document **must** be amended to include these reasons.
+
+When a conflict arises between a maintainer workgroup and a maintainer not part of the work group on a pull request, the same pull request voting procedure can take place.
+However, the votes needed to reach a decision must be a supermajority (66%) instead of a simple majority.
+
+This is intended to allow the workgroup to still achieve their goal or vision if there is friction. However, the entire maintainer team can still weigh in and block or allow a change if they are mostly for or against the change.
+
+Conflicts on pull requests that are internal to the workgroup are subjected to the same PR review procedures; however, these are supposed to be few and far between, with most conflicts being resolved through alternative directions or decisions.
+
+### Design Documents
+Workgroups are encouraged to make design documents for their game area.
```

Comment:
Recommend linking to the template: https://docs.spacestation14.com/en/general-development/game-area-design-doc.html